stripper/ze_doom3_v2.cfg

```diff
@@ -475,7 +498,7 @@ modify:
 	}
 	replace:
 	{
-		"iMagnitude" "45"
+		"iMagnitude" "100"
```

Comment:
This should be combined with the "Fix ZM Skeleton triggering explosion too many times" change. It certainly isn't a nerf on its own anymore considering the magnitude is increased, and doesn't make sense to be separate considering how linked the changes are.